iOS/FVM/FVM/HighlightHandler.swift

```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 Flying Vehicle Monster team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SceneKit
+
+internal class HighlightHandler {
+    private var nodeMaterialMap = Dictionary<String, (node: SCNNode, material: Any)>()
+    private let animationDuration: CFTimeInterval = 0.5
+    
+    internal func setHighlightOn(node: SCNNode) {
+        SCNTransaction.begin()
```

Comment:
@ghawliczek Is that possible to move redundant code to separated method (  SCNTransaction.begin() and   SCNTransaction.commit()) and paste body of func as a argument? (as Closure?)